INSTALL.md

```diff
@@ -1,108 +1,79 @@
 # PyBitmessage Installation Instructions
+- Binary (no separate installation of dependencies required)
+    - windows (32bit only): https://download.bitmessage.org/snapshots/
+    - linux (64bit): https://appimage.bitmessage.org/
+    - mac (64bit, not up to date): https://github.com/Bitmessage/PyBitmessage/releases/tag/v0.6.3
+- Source
+    git clone git://github.com/Bitmessage/PyBitmessage.git
 
-For an up-to-date version of these instructions, please visit the
-[Bitmessage Wiki](https://bitmessage.org/wiki/Compiling_instructions).
+## setuptools
+This is now the recommended and in most cases the easiest procedure for installing PyBitmessage. There is also a helper script to resolve dependencies easily.
 
-PyBitmessage can be run in either one of two ways:
+Go to the directory with PyBitmessage source code and run:
```

Comment:
put checkdeps before setuptools and say it helps if you're installing from source